code/script.js

```diff
@@ -93,14 +37,68 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
   showMessage("Hello there, What's your name?", 'bot');
 }
 setTimeout(greetUser, 1000);
 
 
+const handleNameInput = (event) => {
+  event.preventDefault();
+  const name = nameInput.value 
+  showMessage(`Hi! My name is ${name}`, 'user');
+  nameInput.value=''
+
+  setTimeout( () => showMessage(`Hello ${name}! What are you in the mood for today?`, 'bot'), 800);
+
+  setTimeout(() => WhatToEat(name), 1200)
+}
+//Here I call the next function
+
+const WhatToEat = (name) => {
+
+  //A choice of different food options show up here
+  inputWrapper.innerHTML =
+  `<button id="Pasta" type="submit" value="Pasta">Pasta</button>
+  <button id="Salad" type="submit" value="Salad">Salad</button>
+  <button id="Meat" type="submit" value="Meat">Meat</button>`;
+
+  //Eventlisteners for the previous buttons - these should call for the next function as well
+
+let Pasta = document.getElementById('Pasta');
```

Comment:
You are writing some good code here! Perhaps to keep code clean and follow naming conventions it could be let pasta instead of let Pasta?